pkg/operatorsource/configuring_test.go

```diff
@@ -155,7 +155,10 @@ func TestReconcile_NotConfigured_NewCatalogConfigSourceObjectCreated(t *testing.
 		builders.OpsrcOwnerNameLabel:      "foo",
 		builders.OpsrcOwnerNamespaceLabel: "marketplace",
 	}
-	opsrcIn.SetLabels(labelsWant)
+	labelsNeed := map[string]string{
+		"opsrc-group": "Community",
```

Comment:
Why are you introducing this label?

Reply:
This is a label from a bit higher up https://github.com/operator-framework/operator-marketplace/pull/208/files/3d69194d66021d90a39618fd391a35ccedd95565..58cd8130be68800cfbddd24630df27330eeb5c2f#diff-0a815e1d2edb82030bf32b730e9aba4fR154

Reply:
@galletti94 Could you update this link? It's out of date because of a rebase I assume